opds.py

```diff
@@ -539,7 +539,7 @@ def page(cls, _db, title, url, lane, annotator=None,
         for args in cls.facet_links(annotator, facets):
             OPDSFeed.add_link_to_feed(feed=feed.feed, **args)
 
-        if len(works) > 0:
+        if len(works) > 0 and pagination.has_next_page:
```

Comment:
Yeah, this should be tested on the clients before it's released just in case something crashes when the next link is missing. I do think it's a good change though.